tests/test_tuya_trv.py

```diff
@@ -152,41 +158,68 @@ async def async_success(*args, **kwargs):
             }
         )
         await wait_for_zigpy_tasks()
-        m1.assert_called_with(
+
+        assert m1.call_args_list[0] == mock.call(
             cluster=0xEF00,
             sequence=2,
-            data=sys_mode_msg[Thermostat.SystemMode.Heat],
+            data=sys_mode_msg[Thermostat.SystemMode.Heat][0],
             command_id=0,
             timeout=5,
             expect_reply=False,
             use_ieee=False,
             ask_for_ack=None,
             priority=t.PacketPriority.NORMAL,
         )
+        if m1.call_count == 2:
```

Comment:
Actually, we may want to add another parameter to `parametrize`?
If the quirk is broken and just doesn't turn off the schedule, the `call_count` would be `1` here, so this portion of the test wouldn't run, but the test won't complain, right?